packages/query-engine/source/utils/errors/NotFoundError.ts

```diff
@@ -0,0 +1,26 @@
+import { NextFunction, Response, Request } from "express";
+import { constants } from "@hypertool/common";
+
+const { httpStatuses } = constants;
+
+const NotFoundError = (
+    error: any,
+    request: Request,
+    response: Response,
+    next: NextFunction,
+): void => {
+    if (
+        error.code === "ER_BAD_DB_ERROR" ||
```

Comment:
Are these error codes from Knex? Or did you create them? If you created them, please no abbreviations.

Reply:
Error codes are from SQL. They are SQL-specific.